backend/src/Github/DTO/GithubPRResponseDTO.php

```diff
@@ -33,6 +34,15 @@ public static function fromGQLResponse(array $data): self
             $data['updatedAt'],
             array_unique(array_map(static fn($commit) => $commit['commit']['message'], $data['commits']['nodes'])),
             $data['files'] ?? null,
+            array_map(static fn($review) => [
+                'body' => $review['body'],
+                'comments' => array_map(static fn($comment) => [
```

Comment:
BOT4:
The `startLine` attribute in the comment array may be `null` based on the GitHub API, yet this case is not handled here. This might lead to potential type inconsistencies or errors when `startLine` is not provided. It is advisable to account for the possibility of null values explicitly.